corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -745,7 +745,7 @@ def _record_timing(self, status):
             'device_type:{}'.format('webapps' if is_webapps else 'other'),
         ]
         maybe_add_domain_tag(self.domain, tags)
-        timer_buckets = (5, 20, 60, 120)
+        timer_buckets = (1, 5, 20, 60, 120, 300, 600)
```

Comment:
If these are intended to be the same, and stay the same, perhaps you could pull out the `timer_buckets` as a constant referenced by both?

Reply:
I went to do that, but I didn't know where to define the constant. The uses are so far away from each other.

Some options I considered, but none seemed quite appropriate:
```py
casexml.apps.phone.const  # should receiverwrapper be importing from here?
corehq.apps.receiverwrapper.util  # not really a util, and casexml code should probably not import from here
corehq.util.datadog.metrics  # what to name it if it lives here?
```

Suggestions?